README.md

```diff
@@ -13,20 +13,20 @@ Installers and binaries for both Windows and Linux are available from **[our hom
 # System Requirements
 
 ## Minimum
-* Windows/Linux OS
+* OS: Windows or Linux
```

Comment:
@toehead2001 Yeah, you're right. Good catch. :smile: 

@Blackbird88 To be fair, he is right. Linux is just the kernel, the outlying GNU tools and the rest if the OS is separate from the kernel as they are made by different groups of people.